bom/pom.xml

```diff
@@ -273,6 +281,11 @@
         <artifactId>hono-dispatch-router</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+          <groupId>javax.annotation</groupId>
+          <artifactId>javax.annotation-api</artifactId>
+          <version>${javax.annotation.api.version}</version>
+      </dependency>
```

Comment:
this dependency is not needed when I compile and run integration tests using Java 10